hudi-utilities/src/main/java/org/apache/hudi/utilities/config/HoodieDeltaStreamerConfig.java

```diff
@@ -0,0 +1,283 @@
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "spark_avro_post_processor.enable")
+        .defaultValue(false)
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> DELETE_COLUMN_POST_PROCESSOR_COLUMN_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.delete.columns")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_ADD_COLUMN_NAME_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.add.column.name")
+        .noDefaultValue()
+        .withDocumentation("New column's name");
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_ADD_COLUMN_TYPE_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.add.column.type")
+        .noDefaultValue()
+        .withDocumentation("New column's type");
+
+    public static final ConfigProperty<Boolean> SCHEMA_POST_PROCESSOR_ADD_COLUMN_NULLABLE_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.add.column.nullable")
+        .defaultValue(true)
+        .withDocumentation("New column's nullable");
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_ADD_COLUMN_DEFAULT_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.add.column.default")
+        .noDefaultValue()
+        .withDocumentation("New column's default value");
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_ADD_COLUMN_DOC_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.add.column.doc")
+        .noDefaultValue()
+        .withDocumentation("Docs about new column");
+
+    public static final ConfigProperty<String> SCHEMA_REGISTRY_BASE_URL_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.baseUrl")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SCHEMA_REGISTRY_URL_SUFFIX_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.urlSuffix")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SCHEMA_REGISTRY_SOURCE_URL_SUFFIX = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.sourceUrlSuffix")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SCHEMA_REGISTRY_TARGET_URL_SUFFIX = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.targetUrlSuffix")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SOURCE_SCHEMA_FILE_PROP = ConfigProperty
```

Comment:
Add a comment to indicate the section of configs for `FilebasedSchemaProvider`.  Mention in the docs that the configs apply only to `FilebasedSchemaProvider`.